### ISSUE TITLE
IPTSD systemd service take long time to start than time of executing iptsd directly

### ISSUE THREAD

**Issue by NP-chaonay** [quo/iptsd#2]
Tested on: Ubuntu 22.04 LTS / Linux Surface Kernel / Surface Laptop Studio

Executing iptsd directly got less than a second
While "systemctl start iptsd" got about 1-2 minutes

**Comment by NP-chaonay** [comment]
As I test, it seem to wait dev-ipts-15.device.

And I found that the file in a code tree "/etc/udev/50-ipts.rules" should be added:
```KERNEL=="ithc", TAG+="systemd"```

after I added, the ```systemctl status /dev/ithc``` works

However even I adjust unit file of iptsd service via "systemctl edit" command.
adjust Unit/Wants and Unit/After from dev-ipts-15.device to dev-ithc.device

it have still have a long delay.

in /var/log/syslog said
```Jun  5 13:51:12 NP-chaonay-L1L systemd[1]: dev-ipts-15.device: Job dev-ipts-15.device/start timed out.```
which I think it is delay.

this also happens on "systemctl restart iptsd" and in /var/log/syslog got the same message after iptsd actually start.
